src/sourmash/cli/utils.py

```diff
@@ -67,10 +67,14 @@ def range_limited_float_type(arg):
     return f
 
 
-def add_tax_threshold_arg(parser, default=0.1):
+def add_tax_threshold_arg(parser, containment_default=0.1, ani_default=None):
```

Comment:
should probably set ani_default here to something other than None, yes?

or... hmm, maybe that causes problems later. will evaluate :)

Reply:
yeah, because the presence of an ani-threshold switches estimation to ANI, I didn't add it bc I didn't want to change default functionality.

That said, I think an ANI threshold is better than a containment threshold, so this would be a better default. Since we didn't really advertise this functionality (and it's explicitly stated as "under investigation" in the docs) - do you think we can just change it here? 

Alternatively, I could change the thresholding argument strategy to allow both default values.

Reply:
I think it's fine to leave it this way for now.